Content/Items/Gear/Weapons/Battleaxe/GuardianAngel.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System.Collections.Generic;
+using PathOfTerraria.Content.Projectiles.Melee;
+using PathOfTerraria.Core.Systems;
+using PathOfTerraria.Core.Systems.Affixes;
+using PathOfTerraria.Core.Systems.Affixes.ItemTypes.WeaponAffixes;
+using ReLogic.Content;
+using Terraria.Localization;
+
+namespace PathOfTerraria.Content.Items.Gear.Weapons.Battleaxe;
+
+internal class GuardianAngel : SteelBattleaxe
+{
+	public override float DropChance => 1f;
+	public override bool IsUnique => true;
+	public override string AltUseDescription => Language.GetTextValue("Mods.PathOfTerraria.Items.GuardianAngel.AltUseDescription");
+	public override string Description => Language.GetTextValue("Mods.PathOfTerraria.Items.GuardianAngel.Description");
```

Comment:
Good call on the localization, I did not do that with mine xD

Reply:
I'm in the habit of doing it, it's easy to forget lol